libphal/phal_sbe.C

```diff
@@ -272,7 +271,92 @@ sbeError_t captureFFDC(struct pdbg_target *target)
 	// create ffdc file
 	tmpfile_t ffdcFile(bufPtr.getData(), ffdcLen);
 	return sbeError_t(exception::SBE_CMD_FAILED, ffdcFile.getFd(),
-			  ffdcFile.getPath().c_str());
+			  ffdcFile.getPath());
+}
+
+sbeError_t capturePOZFFDC(struct pdbg_target *target)
+{
+	assert(is_ody_ocmb_chip(target) == true);
+	// get SBE FFDC info
+	bufPtr_t bufPtr;
+	uint32_t ffdcLen = 0;
+	uint32_t status = 0;
+
+	// sbe_ffdc_get internall uses chipop target so probe it
+	pdbg_target *co = get_ody_chipop_target(target);
```

Comment:
Try to avoid copying by doing `if (pdbg_target_probe(get_ody_chipop_target(target)) != PDBG_TARGET_ENABLED)` if possible